services/debug_tools.py

```
@@ -0,0 +1,35 @@
+
+These helpers are intentionally permissive and should not be enabled in
+production.
+"""
+
+import json
+from urllib.request import urlopen
+
+from db import MessageRepository, UserProfileRepository
+
+
+ADMIN_DEBUG_TOKEN = "alain-admin-export-2026"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "ADMIN_DEBUG_TOKEN"

(S105)